src/framework/parsers/sogs.js

```diff
@@ -61,6 +97,11 @@ class SogsParser {
     }
 
     async loadTextures(url, callback, asset, meta) {
+        // transform meta to latest shape
+        if (meta.version !== 2) {
+            meta = upgradeMeta(meta);
```

Comment:
do we want to add some Debug.deprecated call to warn people they use deprecated format? 
Or do we officially support both in the future.